src/lib.rs

```diff
@@ -33,6 +33,7 @@
 #![cfg_attr(test, deny(warnings))]
 #![cfg_attr(test, feature(test))]
 #![allow(trivial_numeric_casts)]
+#![feature(test)]
```

Comment:
Unfortunately this change means that this library cannot compile on the stable compiler, but it just means that all benchmarking-related code should be behind a `#[cfg(test)]` attribute (e.g. the module containing the benchmarks should have `#[cfg(test)]`